__tests__/library/commands.test.ts

```diff
@@ -1,16 +1,43 @@
+import { ICommandClasses } from '../../src/library/commandLoader';
 import Commands from '../../src/library/commands';
+import ICommand from '../../src/library/iCommand';
 
-// TODO: I feel like this class is too basic to test,
-//  and ultimately a wrapper for other classes that have been tested
-//  Might just remove it some day
 describe('Commands', () => {
-  const commands = new Commands();
-  test('Has a command', () => {
-    expect(Object.keys(commands.all).length).toBeGreaterThan(0);
-    expect(commands.names.length).toBeGreaterThan(0);
+  let mockHelloCommand: ICommand;
+  let mockYetAnotherCommand: ICommand;
+  let mockCommands: ICommandClasses;
+  let commands: Commands;
+
+  beforeEach(() => {
+    // TODO: these should probably go into a factory/mock
+    mockHelloCommand = {
+      name: 'Hello',
+      description: 'Hello World',
+      execute: jest.fn()
+    };
+    mockYetAnotherCommand = {
+      name: 'YAC',
+      description: 'Yet Another Command!',
+      execute: jest.fn()
+    };
+    mockCommands = {
+      hello: mockHelloCommand,
+      yac: mockYetAnotherCommand
+    };
+    commands = new Commands(mockCommands);
+  });
+
+  test('.names returns command names', () => {
+    const commandNames = ['hello', 'yac'];
+    expect(commands.names).toEqual(commandNames);
```

Comment:
Maybe I should have a test block that says that command names are transformed. I would have to double check the code, but I think that they are turned into lower case, and are converted from kebab case to camel case if I recall correctly.

Maybe those tests belong in a different test or are already tested,, would be good to verify.

Ultimately it may be implicitly tested here, but I would like a block to explicitly state the business logic behind it for the sake of documentation.